server/src/cores/storage.core.ts

```diff
@@ -105,6 +105,14 @@ export class StorageCore {
     return originalPath.startsWith(StorageCore.getBaseFolder(StorageFolder.ENCODED_VIDEO));
   }
 
+  static isAbsolutePath(path: string) {
+    if (!path.startsWith('/')) {
+      return false;
+    }
+
+    return path === resolve(path);
+  }
```

Comment:
Small nit: You can use a built in method for this: https://nodejs.org/api/path.html#pathisabsolutepath

Reply:
Done

Reply:
Uhm is it?

Reply:
I probably hadn't pushed yet. Should be fine now!